src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/ExceptionsAndErrors.java

```diff
@@ -59,6 +59,8 @@
 
   CLUSTER_APPLIER_SERVICE_STATS_COLLECTOR_ERROR("ClusterApplierServiceStatsCollector"),
 
+  MASTER_CLUSTER_UPDATE_STATS_COLLECTOR_ERROR("MasterClusterStateUpdateStatsCollector"),
```

Comment:
Can we rename it to "MasterClusterStateUpdateStatsCollectorError"?

Reply:
Other constants have been taken care in the other PR:
https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/pull/583/files